packages/pulpcore/python-pulp_2to3_migration/python-pulp_2to3_migration.spec

```diff
@@ -23,8 +23,8 @@ BuildRequires:  python3-setuptools
 Summary:        %{summary}
 %{?python_provide:%python_provide python3-%{pypi_name}}
 
-Requires:       python3-pulpcore >= 3.3
-Requires:       python3-jsonschema
+Requires:       python3-pulpcore >= 3.4
+Requires:       python3-jsonschema >=3.0
```

Comment:
need a space after the '=' i believe :)

Reply:
don't tell me how to creatively break RPM spec files! ;-) (fixed!)